sda-auth/frontend/static/login.js

```diff
@@ -2,11 +2,8 @@ $.get('/login-options', (options) => {
     options.forEach(option => {
         $("#login-options").append(
             `<div class="col-sm py-3">
-      <p class="lead">
-        For logging in with ${option.Name} please click here:
-      </p>
-      <a class="btn btn-primary btn-lg btn-block" href="${option.URL}">
-        Log in with ${option.Name}
+      <a href="${option.URL}">
+        <img src="public/${option.Name}.png" class="img-fluid">
```

Comment:
I have a slight preference for implementing these in CSS instead, but this obviously works.

Reply:
The entire UI should be redesigned by someone who can do it properly, more CSS and less java